lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_add_testers_action.rb

```diff
@@ -0,0 +1,78 @@
+require 'fastlane/action'
+require 'fastlane_core/ui/ui'
+
+require_relative '../helper/firebase_app_distribution_helper'
+require_relative '../helper/firebase_app_distribution_auth_client'
+
+module Fastlane
+  module Actions
+    class FirebaseAppDistributionAddTestersAction < Action
+      extend Auth::FirebaseAppDistributionAuthClient
+      extend Helper::FirebaseAppDistributionHelper
+
+      def self.run(params)
+        auth_token = fetch_auth_token(params[:service_credentials_file], params[:firebase_cli_token])
+        fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token, params[:debug])
+
+        if blank?(params[:emails]) && blank?(params[:file])
+          UI.user_error!("Must specify `emails` or `file`.")
+        end
+
+        emails = get_value_from_value_or_file(params[:emails], params[:file]).split(',')
+
+        if emails.count > 1000
+          UI.user_error!("A maximum of 1000 testers can be added at a time.")
+        end
+
+        fad_api_client.add_testers(params[:project], emails)
+
+        UI.success("✅ Tester(s) successfully added.")
+      end
+
+      def self.description
+        "Create testers"
+      end
+
+      def self.authors
+        ["Tunde Agboola"]
+      end
+
+      # supports markdown.
+      def self.details
+        "Create testers"
```

Comment:
Maybe some more details here, like "Create testers in bulk from a comma-separated list or a file." or even more detail? Maybe a link to docs if there is one?

Reply:
Done.

Reply:
@tagboola you updated `self.description` but not `self.details` :)

I think if one is going to be more detailed it should be `details`.

Reply:
Ahh my bad. Updated `details` as well.